index.html

```diff
@@ -0,0 +1,211 @@
+                        <div>Summit 2015</div>
+                    </div>
+        </div>
+        <div class="red">
+            <h3>See the past Fitness Summits</h3>
+            <div class="line l-3"></div>
+            <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <div class="red-img">
+                <img src="images/1679012436849.jpg" alt="">
+                <img src="images/1679012436857.jpg" alt="">
+            </div>
+        </div>
+        <footer class="last">
+            <strong class="logo"><img src="images/fitness_center_FILL0_wght400_GRAD0_opsz48.svg" alt=""> Fitness-G</strong>
+            <p>qwerty 4, 4322 Norway (love what we do) <br> fit-G@gmail.com <br>055872787</p>
+        </footer>
+    </section>
+</section>
+<section class="mobile">
+    <div id="mobile-home">
+        <div class="top">
+    <button class="hamburger" type="submit">
+        <div class="bar"></div>
+        <nav class="mobile-nav">
+            <a class="take" href="#mobile-about">About</a>
+            <a>Home</a>
+        <a>Join</a>
+        <a>Sponsor</a>
+        <a>News</a>
+        </nav>
+    </button>
+        <h2>"Health is Wealth"</h2>
+        <h1>COME TRAIN WITH US  <br> BE THE BEST YOU...</h1>
+        <div class="box">Fitness is not just about physical health, it's about mental strength, self-discipline, and the feeling of accomplishment that comes with pushing yourself beyond your limits. Start your journey today!</div>
+        <h3>Open: 24/7(MON)-(FRI)</h3>
+        <p>@Green square. Art Centre Goal and more</p>
+        </div>
+        <div class="main">
+            <h2>Main Program</h2>
+            <div class="line"></div>
+            <div class="big-box">
+                <div class="inner-box"><img src="images/program_icon_01 1.png" alt=""><h4>Lecture</h4><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+                <div class="inner-box"><img src="images/program_icon_02 1.png" alt=""><h4>CC Exhibition</h4><p>Appreciate various creations applying CC license of artist, organized from the art center Goal</p></div>
+                <div class="inner-box"><img src="images/program_icon_03 1.png" alt=""><h4>Forum</h4><p>Have the time to share your thoughts and opinions with experts for each topic.</p></div>
+                <div class="inner-box"><img src="images/program_icon_04 1.png" alt=""><h4>Workshop</h4><p>Try creating your own work using open source license rather than just watching at it.</p></div>
+                <div class="inner-box"><img src="images/program_icon_05 1.png" alt=""><h4>CC Ignite</h4><p>get opportunities to network with CC affiliates around the world, also after the summit.</p></div>
+            </div>
+            <button>Join The Fitness Summit 2015</button>
+        </div>
+        <div class="speakers">
+            <h1>Featured Speakers</h1>
```

Comment:
- [ ] For [Semantic purposes](https://www.w3schools.com/html/html5_semantic_elements.asp), there should be only one `h1` in the page. Using only one H1 creates a hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Kindly consider using other `hx` HTML tags ☺️.